node v4 support

Will need to fix this:

```
ok 65 fixture load
_http_client.js:52
    throw new Error('Protocol "' + protocol + '" not supported. ' +
    ^

Error: Protocol "https:" not supported. Expected "http:".
    at new ClientRequest (_http_client.js:52:11)
    at Object.exports.request (http.js:31:10)
    at Object.socket.once.exports.request (https.js:182:15)
    at Get.request (/Users/dane/projects/mapbox-tile-copy/node_modules/tilejson/node_modules/get/lib/node-get/node-get.js:92:22)
    at Get.perform (/Users/dane/projects/mapbox-tile-copy/node_modules/tilejson/node_modules/get/lib/node-get/node-get.js:132:30)
    at Get.asBuffer (/Users/dane/projects/mapbox-tile-copy/node_modules/tilejson/node_modules/get/lib/node-get/node-get.js:286:10)
    at TileJSON.get (/Users/dane/projects/mapbox-tile-copy/node_modules/tilejson/lib/tilejson.js:207:19)
    at TileJSON.getTile (/Users/dane/projects/mapbox-tile-copy/node_modules/tilejson/lib/tilejson.js:138:10)
    at getTileRetry (/Users/dane/projects/mapbox-tile-copy/node_modules/tilelive/lib/stream-util.js:247:12)
    at get (/Users/dane/projects/mapbox-tile-copy/node_modules/tilelive/lib/stream-scanline.js:103:9)
npm ERR! Test failed.  See above for more details.
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -2,7 +2,6 @@ language: node_js
 node_js:
 - '0.10'
 - 4
-- 6
 sudo: false
 before_install:
 - if [[ $(uname -s) == 'Linux' ]]; then wget https://launchpad.net/~ubuntu-toolchain-r/+archive/ubuntu/test/+files/libstdc%2B%2B6_4.8.1-2ubuntu1~12.04_amd64.deb && dpkg -x libstdc++6_4.8.1-2ubuntu1~12.04_amd64.deb ./ && export LD_PRELOAD=$(pwd)/usr/lib/x86_64-linux-gnu/libstdc++.so.6; fi

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,6 +1,8 @@
 language: node_js
 node_js:
 - '0.10'
+- 4
+- 6
 sudo: false
 before_install:
 - if [[ $(uname -s) == 'Linux' ]]; then wget https://launchpad.net/~ubuntu-toolchain-r/+archive/ubuntu/test/+files/libstdc%2B%2B6_4.8.1-2ubuntu1~12.04_amd64.deb && dpkg -x libstdc++6_4.8.1-2ubuntu1~12.04_amd64.deb ./ && export LD_PRELOAD=$(pwd)/usr/lib/x86_64-linux-gnu/libstdc++.so.6; fi

